lib/Common/Memory/Recycler.h

```diff
@@ -1146,6 +1146,8 @@ class Recycler
 #ifdef NTBUILD
     void SetTelemetryBlock(RecyclerWatsonTelemetryBlock * telemetryBlock) { this->telemetryBlock = telemetryBlock; }
 #endif
+    
+    uint GetPinnedObjectCount() const { return this->pinnedObjectMap.Count(); }
```

Comment:
Is this used anywhere? Maybe in Full? Is there a corresponding Full review?

Reply:
It is used in Full. I will make a PR once Full passes tests. 
These are existing changes though. They merged just after we forked 1.10, but we want them there.